sdks/python/apache_beam/utils/retry.py

```diff
@@ -209,7 +209,6 @@ def wrapper(*args, **kwargs):
             # Traceback objects in locals can cause reference cycles that will
             # prevent garbage collection. Clear it now since we do not need
             # it anymore.
-            sys.exc_clear()
```

Comment:
`sys.exc_clear` has been removed in Python 3.
Reading the [docs](https://docs.python.org/2/library/sys.html#sys.exc_info), this call does not seem necessary as long as the local variable with reference to the traceback is removed.